spec/fabricators/video_fabricator.rb

```diff
@@ -0,0 +1,5 @@
+Fabricator(:video) do 
+  title {Faker::Lorem.word}
+  description {Faker::Lorem.paragraph}
+  category {Fabricate(:category)}
```

Comment:
You don't need the block here. You can have just `category` on this line and the gem is smart enough to make the association and fabricate an object for you.

http://www.fabricationgem.org/#defining-fabricators

Scroll down a bit to "associations"